cmd/main.go

```diff
@@ -1 +1,34 @@
-package cmd
+package main
+
+import (
+	"fmt"
+	"log"
+	"todo-clean/database"
+	"todo-clean/delivery"
+	"todo-clean/repository"
+	"todo-clean/usecase"
+
+	"github.com/gin-gonic/gin"
+)
+
+func main() {
+
+	dbConn, err := database.ConnectDB()
+	//
+	if err != nil {
+		log.Panicln("failed to connect database", err)
+	} else {
+		fmt.Println("Connect ok", dbConn)
+	}
+	engine := gin.New()
+
+	repo := repository.NewRepository(dbConn)
+	use := usecase.NewUseCase(repo)
+	//
+	delivery.NewHandler(engine, use)
+	//
+	err = engine.Run("localhost:8080")
```

Comment:
please use port from docker env

Reply:
i use it because have some problem with docker file